docs/feature_split_keyboard.md

```diff
@@ -23,7 +23,17 @@ Alternatively, you can use any sort of cable and socket that has at least 3 wire
 
 If you want to use I<sup>2</sup>C to communicate between halves, you will need a cable with at least 4 wires and 2x 4.7kΩ pull-up resistors.
 
+#### Considerations 
 
+The most commonly used connection is a TRRS cable and jacks.  These provide 4 wires, making them very useful for split keyboards, and are easy to find. 
+
+However, since one of the wires carries VCC, this means that the boards are not hot pluggable. You should always disconnect the board from USB before unplugging and plugging in TRRS cables, or you can short the controller, or worse. 
```

Comment:
What's a typical TRRS pinout look like? Are you telling me VCC is not on the sleeve? 😞

Reply:
there's at least 2 variants. I know that  keebio changed  there's to something more logical or less likely to short. @nooges any insight you'd like to share?

Reply:
I also just realised that even if VCC were on the sleeve you would still need to (un)plug the slave side first, otherwise you would be sending it VCC through the data or GND pins.

Reply:
Yup, exactly :(

This happens with my crkbd, and actually crashes the USB hub.  (my monitor) :(

And the way that @nooges has his set up, it's not as much of an issue, IIRC. But it can still happen.

Reply:
Data lines are able to tolerate being shorted to vcc temporarily (especially when ground isn't present yet.)

The big issue with using a TRRS plug is that adjacent rings get temporarily shorted together as it is inserted, and if vcc and ground are kept on tip and sleeve (i.e. separated by one or more rings) then they should almost always stay isolated. Of course, it's quite likely inexperienced people will decide to do data-data-vcc-ground... D:

Reply:
@pelrun I hate to ask, but would you mind expanding upon that and adding that info here (or as a PR to my repo for this branch)?